sdp.c

```diff
@@ -600,6 +600,7 @@ int janus_sdp_anonymize(janus_sdp *anon) {
 		} else if(m->type == JANUS_SDP_VIDEO && m->port > 0) {
 			video++;
 			m->port = video == 1 ? 9 : 0;
+#ifdef HAVE_SCTP
```

Comment:
I think I had this ifdef removed because it resulted in missing media lines in case data channels support was not compiled. Or more likely this was happening in the merge method... anyway, whatever the case I'm not sure it's needed here, as the merge process sets the port to 0 anyway if you don't have DCs.